README.md

```diff
@@ -1,2 +1,36 @@
 # TinderRecipe
- 
+###Project description
+This project contains the back end for a handy and useful 
+multiplatform application for people who are enthusiastic for cooking,
+love sharing and trying new recipes. The mobile application is based
+on Tinder app. Users will be able to swipe through the pictures of
+recipes, and respectively like or dislike them.
+When a recipe is liked, it will be saved in user's favorite recipes, 
+and there they can look at it later. 
+
+
+###Languages and tools
+* ***Kotlin*** - *main coding language*
+* ***PostgreSQL*** - *the free open-source database that we use*
+* ***GraphQL*** - *an open-source data query and manipulation language for APIs*
+
```

Comment:
Exposed for database migration and connection is not mentioned 
Ktor (the server backend) is not mentioned as well

Reply:
Done